integ/components/deadline/deadline_02_renderQueue/bin/deadline_02_renderQueue.ts

```diff
@@ -27,17 +27,20 @@ const storage = new StorageStruct(componentTier, 'StorageStruct', {
   useMongoDB: false,
 });
 
-// Create test struct for Render Queue in http mode
-const render1 = new RenderStruct(componentTier, 'RenderStructRQ1', {
-  integStackTag,
-  repository: storage.repo,
-  protocol: 'http',
-});
-//Create test struct for Render Queue in https mode
-const render2 = new RenderStruct(componentTier, 'RenderStructRQ2', {
-  integStackTag,
-  repository: storage.repo,
-  protocol: 'https',
-});
+const structs: Array<RenderStruct> = [];
```

Comment:
Again, I think we can just initialize without `push`.

Reply:
Fixed.